### ISSUE TITLE
Error while launching container  inside of the SysBox System Container

### ISSUE THREAD

**Issue by Deepansharora27** [nestybox/sysbox#630]
Hi Team, 

I have been trying to deploy a System Container with Systemd, sshd, and Docker inside. Although I have been successfully able to deploy this system container. When I try to run my dockerized image(or any Docker Image for that matter), it returns me the following error: 

`docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: unable to apply cgroup configuration: mkdir /sys/fs/cgroup/rdma/docker: permission denied: unknown`

Correct me if I am wrong but this error is being triggered because the SysBox Container does not have the necessary settings to modify the `cgroup` settings. 

Another thing I would like to highlight here is that, I am testing out SysBox inside of a GCP VM and I am set to the Project Owner as well. So I believe spinning up the Container inside of a VM is not the cause of the issue here. 

The VM OS Distribution is Ubuntu 18.04.6 LTS and I have checked that this Linux  Distribution is supported by SysBox. 

Your Help would be appreciated on this 
Thanks

**Comment by ctalledo** [comment]
Hi @Deepansharora27, thanks for trying Sysbox, hope you find it useful.

> When I try to run my dockerized image(or any Docker Image for that matter), it returns me the following error
> `docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: unable to apply cgroup configuration: mkdir /sys/fs/cgroup/rdma/docker: permission denied: unknown`

Is that error reported by the Docker engine inside the Sysbox container, or by the Docker engine at host level?

**Comment by Deepansharora27** [comment]
Hi @ctalledo 
It is being reported by the Docker Engine inside the SysBox Container.

Well to reproduce, it is giving me the issue with any docker image, not a specific docker image

Even if I run 
`sudo docker run hello-world` 

It results me the same error

**Comment by Deepansharora27** [comment]
@ctalledo Hi, were you able to reproduce this at your End ?

**Comment by ctalledo** [comment]
hi @Deepansharora27, no I am not able to repro.

Are you running the latest version of Sysbox (v0.5.2)? Sounds like you may be running an older version, per issue #560 .

**Comment by Deepansharora27** [comment]
I sourced the latest binary, but let me check once again

**Comment by Deepansharora27** [comment]
I just checked it was the `0.5.0` version, but here in the Installation Guide https://github.com/nestybox/sysbox/blob/master/docs/user-guide/install-package.md
It only mentions 0.5.0 version while trying to install it via `wget`

**Comment by Deepansharora27** [comment]
While Installing the SysBox 0.5.2 version, I get the following error: 

`The Sysbox installer requires a docker service restart to configure network parameters, but it cannot proceed due to existing Docker containers. Please remove them as indicated below and re-launch the installation process. Refer to Sysbox installation documentation for details.
        "docker rm $(docker ps -a -q) -f"

dpkg: error processing package sysbox-ce (--configure):
 installed sysbox-ce package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 sysbox-ce
E: Sub-process /usr/bin/dpkg returned an error code (1)`


Then, I had a look at the Troubleshoot section, and it mentions to execute 
`systemctl list-units -t service --all | grep sysbox`

But it results me a Blank output, and does not show me the service running as well

**Comment by ctalledo** [comment]
Hi @Deepansharora27, try stopping all running Docker containers and the run the installer again please.

**Comment by rodnymolina** [comment]
@Deepansharora27, the installation error logs above explain what the issue is and how to fix it. Please run the suggested instruction and try to install it again:

"docker rm $(docker ps -a -q) -f"

**Comment by Deepansharora27** [comment]
Hi @rodnymolina 

I ran this but this results into an error

**Comment by Deepansharora27** [comment]
@rodnymolina This is what I get while executing `docker rm $(docker ps -a -q) -f`

`
permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1": dial unix /var/run/docker.sock: connect: permission denied
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
`

**Comment by Deepansharora27** [comment]
@rodnymolina This is what I get while executing`sudo docker rm $(docker ps -a -q) -f`

This is the Error returned:

`
permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json?all=1": dial unix /var/run/docker.sock: connect: permission denied
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
`

**Comment by rodnymolina** [comment]
@Deepansharora27, that's right, the docker instruction above returns an error when there are no existing containers (we should provide a slightly different one to avoid any confusion), so you should be able to re-install sysbox now with the same `apt-get install` instruction you previously used. Are you saying that you are still getting the same installation error?
